fronts-client/src/components/card/recipe/RecipeCard.tsx

```diff
@@ -63,27 +64,48 @@ export const RecipeCard = ({
   const paths = recipe?.canonicalArticle
     ? getPaths(recipe.canonicalArticle)
     : undefined;
+  const isContentNotFound = recipe?.title ? false : true;
 
   return (
     <CardContainer {...rest}>
       <CardBody data-testid="snap" size={size} fade={fade}>
         {showMeta && (
-          <CardMetaContainer size={size}>
-            <CardMetaHeading>Recipe</CardMetaHeading>
+          <CardMetaContainer size={size} isContentNotFound={isContentNotFound}>
+            {recipe?.title ? (
+              <CardMetaHeading>Recipe</CardMetaHeading>
+            ) : (
+              <img
+                src={noRecipeIcon}
+                style={{
+                  position: 'relative',
+                  width: '50%',
+                  height: '50%',
+                  top: '20%',
+                  left: '25%',
+                }}
+              />
+            )}
             <CardMetaContent>
               {upperFirst(recipe?.difficultyLevel)}
             </CardMetaContent>
           </CardMetaContainer>
         )}
-        <CardContent textSize={textSize}>
+        <CardContent textSize={textSize} isContentNotFound={isContentNotFound}>
           <CardHeadingContainer size={size}>
-            <CardHeading data-testid="headline" html>
-              {recipe?.title ?? 'No recipe found'}
+            <CardHeading
+              data-testid="headline"
+              html
+              isContentNotFound={isContentNotFound}
+            >
+              {recipe?.title ??
+                'This recipe won’t load in the app, please select an alternative.'}
```

Comment:
Suggested wording change - from "won't load" to "might not load", as we can't be 100% sure it's not a bug on our side 🤭